EIPS/eip-4974.md

```diff
@@ -50,63 +50,23 @@ pragma solidity ^0.8.0;
-    event Participation(address indexed _participant, bool _participation);
-
-    /// @dev Emits when operator transfers EXP. 
-    ///  MUST emit when EXP is transferred by any mechanism.
-    ///  MUST ONLY emit by `transfer`.
-    event Transfer(address indexed _from, address indexed _to, uint256 _amount);
-
-    /// @notice Appoint operator authority.
-    /// @dev MUST throw unless `msg.sender` is `operator`.
-    ///  MUST throw if `operator` address is either already current `operator`
-    ///  or is the zero address.
-    ///  MUST emit an `Appointment` event.
-    /// @param _operator New operator of the smart contract.
-    function setOperator(address _operator) external;
-
-    /// @notice Activate or deactivate participation. CALLER IS RESPONSIBLE TO
-    ///  UNDERSTAND THE TERMS OF THEIR PARTICIPATION.
-    /// @dev MUST throw unless `msg.sender` is `participant`.
-    ///  MUST throw if `participant` is `operator` or zero address.
-    ///  MUST emit a `Participation` event for status changes.
-    /// @param _participant Address opting in or out of participation.
-    /// @param _participation Participation status of _participant.
-    function setParticipation(address _participant, bool _participation) external;
-
-    /// @notice Transfer EXP from one address to a participating address.
-    /// @dev MUST throw unless `msg.sender` is `operator`.
-    ///  MUST throw unless `to` address is participating.
-    ///  MUST throw if `to` and `from` are the same address.
-    ///  MUST emit a Transfer event with each successful call.
-    ///  SHOULD throw if `amount` is zero.
-    ///  MAY allow minting from zero address, burning to the zero address, 
-    ///  transferring between accounts, and transferring between contracts.
-    ///  MAY limit interaction with non-participating `from` addresses.
-    /// @param _from Address from which to transfer EXP tokens.
-    /// @param _to Address to which EXP tokens at `from` address will transfer.
-    /// @param _amount Total EXP tokens to reallocate.
-    function transfer(address _from, address _to, uint256 _amount) external;
-
-    /// @notice Return total EXP managed by this contract.
-    /// @dev MUST sum EXP tokens of all `participant` addresses, 
-    ///  regardless of participation status, excluding only the zero address.
-    function totalSupply() external view returns (uint256);
-
-    /// @notice Return total EXP allocated to a participant.
-    /// @dev MUST register each time `Transfer` emits.
+    /// @dev Emits when operator issues a rating. 
+    ///  MUST emit when rating is assigned by any mechanism.
+    ///  MUST ONLY emit by `rate`.
+    event Rating(address _rated, int8 _rating_);
+
+    /// @notice Rate an address.
+    ///  MUST emit a Rating event with each successful call.
+    /// @param _rated Address to be rated.
+    /// @param _rating Total EXP tokens to reallocate.
+    function rate(address _rated, int8 _rating_) external;
```

Comment:
This is on purpose! 

- It's signed:  Reviewers should be able to give neutral and negative ratings for the wallets and contracts they interact with.
- It's 8bit:  The objective here is to keep ratings within some fathomably comparable range. Longer term, this could encourage easy aggregation of ratings, versus using larger numbers where users might employ a great variety of scales.

Does that make sense?